client/me/purchases/manage-purchase/payment-method-selector/style.scss

```diff
@@ -51,3 +53,23 @@
 	font-size: $font-body-small;
 	font-style: italic;
 }
+
+/* Jetpack cloud overwrites */
+
+.payment-method-selector__content.is-jetpack-cloud {
```

Comment:
This is not ideal, but at the moment it seems the best way to style `composite-checkout` components in Jetpack cloud. Since they use CSS modules, we can't rely on the exact class names.

Reply:
This is fine, but the `composite-checkout` components styles are controlled by a theme that you can change. For example, here's where we change the theme to jetpack colors in checkout:

https://github.com/Automattic/wp-calypso/blob/21f7f7b865bb2aa5a210289115c6ce0f04f71c6d/client/my-sites/checkout/composite-checkout/composite-checkout.tsx#L495-L507

https://github.com/Automattic/wp-calypso/blob/21f7f7b865bb2aa5a210289115c6ce0f04f71c6d/client/my-sites/checkout/composite-checkout/composite-checkout.tsx#L792

You could specify a theme for the `CheckoutProvider` in purchase management in the same way:

https://github.com/Automattic/wp-calypso/blob/21f7f7b865bb2aa5a210289115c6ce0f04f71c6d/client/me/purchases/manage-purchase/payment-method-selector/index.tsx#L120